test/elvis_SUITE.erl

```diff
@@ -312,26 +279,33 @@ check_first_line_output(Fun, Expected, FilterFun) ->
     ct:capture_start(),
     Fun(),
     ct:capture_stop(),
-    Lines = case ct:capture_get([]) of
-                [] -> [];
-                [Head | _] -> [Head]
-            end,
+    Lines =
+        case ct:capture_get([]) of
+            [] ->
+                [];
+            [Head | _] ->
+                [Head]
+        end,
     ListFun = fun(Line) -> FilterFun(Line, Expected) end,
     [_ | _] = lists:filter(ListFun, Lines).
 
 starts_with(Result, Expected) ->
     case string:str(Result, Expected) of
-        1 -> true;
-        _ -> {Expected, Expected} == {Result, Expected}
+        1 ->
+            true;
+        _ ->
+            {Expected, Expected} == {Result, Expected}
     end.
```

Comment:
```suggestion
    string:str(Result, Expected) == 1 orelse {Expected, Expected} == {Result, Expected}.
```

Reply:
Sure. I didn't touch "actual code", though, except where static analysis told me too, but I can change this...